app/src/main/java/com/bootcamp_android/parking_app/adapter/DataViewHolder.kt

```diff
@@ -31,20 +30,24 @@ class DataViewHolder(private val view: View) : RecyclerView.ViewHolder(view) {
     }
 
     fun bindReservation(reservation: DataModel.Reservation) {
+        val start_date = view.findViewById<TextView>(R.id.text_date_start)
+        val star_hour = view.findViewById<TextView>(R.id.text_hour_start)
+        val end_date = view.findViewById<TextView>(R.id.text_date_end)
+        val end_hour = view.findViewById<TextView>(R.id.text_end_hour)
```

Comment:
Do not use underscores in the variable names

Reply:
ok